.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @pvallone @cameronwaterman
```

Comment:
@rajsite @mollykreis let me know if y'all have any suggestions for folks to add here.

Reply:
@atmgrifter00 and I (@rajsite) should be added 👍

Reply:
pushed that change to the branch